README.md

```diff
@@ -1,2 +1,321 @@
-# Quantum Network Explorer CLI
-Command Line Interface to interact with the Quantum Network Explorer
+# Quantum Network Explorer ADK
+Application Development Kit to interact with the Quantum Network Explorer. With the QNE-ADK you can create your application and experiment and run it locally using the commands. 
+
+## Prerequisites
+- A modern Linux 64-bit (x86_64) operating system. If you don’t have linux you could run it via virtualization, e.g. using VirtualBox. If you have Windows 10 you can also use the [Bash on Ubuntu](https://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/) subsystem.
```

Comment:
use Linux with capital L consistently (check rest of document please).

We should refer to an official Microsoft site for WSL: e.g. https://docs.microsoft.com/en-us/windows/wsl/
also Windows 11 is oke!